spec/features/user_adds_coverage_to_a_course_spec.rb

```diff
@@ -22,9 +22,59 @@
     end
   end
 
+  scenario "user adds additional coverage to a course
+  from the coverages page by clicking on an unmatched outcome" do
+    unmatched_outcome = create(:outcome,
```

Comment:
I'd try to limit the setup code as much as possible here. I think maybe you can do something like:

```
course = create(:course)
matched_outcome, unmatched_outcome = create_pair(:outcome, course: course)
coverage = create(:coverage, course: course, outcomes: [matched_outcome])
user = user_with_assessments_access_to(course.department)
```